### PR TITLE
feat(logging): separate fields

### PR DESCRIPTION
**Pull request by neil-sciberras**

### CODE REVIEW

File: src/FluentlyHttpClient/Middleware/LoggerHttpMiddleware.cs [hunk 1]
```diff
@@ -68,30 +71,30 @@ public async Task<FluentHttpResponse> Invoke(FluentHttpMiddlewareContext context
...
 			if (response.Content == null || !(options.ShouldLogDetailedResponse ?? false))
 			{
-				_logger.LoggerHttp_Response(response, stopwatchElapsed.TotalMilliseconds);
+				_logger.LoggerHttp_Response(request.Method, request.Uri!, path, response.StatusCode, stopwatchElapsed.TotalMilliseconds, httpVersion);
 				return response;
 			}
 
 			var responseContent = await response.Content.ReadAsStringAsync();
-			_logger.LoggerHttp_ResponseDetailed(response, response.Headers.ToFormattedString(), responseContent, stopwatchElapsed.TotalMilliseconds);
+			_logger.LoggerHttp_ResponseDetailed(request.Method, request.Uri!, path, response.StatusCode, response.Headers.ToFormattedString(), responseContent, stopwatchElapsed.TotalMilliseconds, httpVersion);
```
Comment (stephenlautier) [inline src/FluentlyHttpClient/Middleware/LoggerHttpMiddleware.cs:97]: httpVersion theres one in the request and one in the response - for this one we can log the response one

File: src/FluentlyHttpClient/Middleware/LoggerHttpMiddleware.cs [hunk 2]
```diff
@@ -166,18 +169,53 @@ public static FluentHttpClientBuilder UseLogging(this FluentHttpClientBuilder bu
...
-
-	[LoggerMessage(LogLevel.Information, "Pre-request... {request}\nHeaders: {headers}\nContent: {requestContent}")]
-	internal static partial void LoggerHttp_RequestDetailed(this ILogger logger, FluentHttpRequest request, string headers, string? requestContent);
-
-	[LoggerMessage(LogLevel.Information, "Post-request... {response} in {elapsed:n0}ms")]
-	internal static partial void LoggerHttp_Response(this ILogger logger, FluentHttpResponse response, double elapsed);
-
-	[LoggerMessage(LogLevel.Information, "Post-request... {response}\nHeaders: {headers}\nContent: {responseContent} in {elapsed:n0}ms")]
-	internal static partial void LoggerHttp_ResponseDetailed(this ILogger logger, FluentHttpResponse response, string headers, string? responseContent, double elapsed);
+	[LoggerMessage(LogLevel.Information, "HTTP request [{method}] '{requestUrl}' ({path}) responded {statusCode:D} in {elapsed:n0}ms ({httpVersion})")]
```
Comment (stephenlautier) [inline src/FluentlyHttpClient/Middleware/LoggerHttpMiddleware.cs:172]:
```suggestion
	[LoggerMessage(LogLevel.Information, "HTTP request ({httpVersion}) [{method}] '{requestUrl}' ({path}) responded {statusCode:D} in {elapsed:n0}ms")]
```

Comment (neil-sciberras) [reply src/FluentlyHttpClient/Middleware/LoggerHttpMiddleware.cs]: done

File: src/FluentlyHttpClient/Middleware/LoggerHttpMiddleware.cs (hunk 1 above)
Comment (neil-sciberras) [reply src/FluentlyHttpClient/Middleware/LoggerHttpMiddleware.cs]: done

**Pull request merged**